scripts/partition_manager/partition_manager.rst

```diff
@@ -38,6 +43,27 @@ Container partitions
 
 The start addresses and sizes of image partitions are used in the preprocessing of the linker script for each image.
 
+RAM partition types
+=====================
+
+Dynamic image RAM partition (DIRP)
+   DIRP is the default RAM partition associated with an image.
+   A DIRP use all RAM which is not defined as static image RAM partitions, or placeholder RAM partitions.
+   A DIRP is only reserved while the image itself is running.
+   Hence, when an image boots the next in the bootloader chain, its DIRP is no longer reserved, and will be used as the DIRP for the next image.
+   All images has a DIRP assigned to it by default.
+   The alternative to DIRP is Static Image RAM partitions (SIRP) described
+   below.
+
+Static Image RAM Partition (SIRP)
```

Comment:
Can an image have both an SIRP and DIRP? I'm assuming not, but I'm not sure. Either way, this could be explained more thoroughly in the docs.

Reply:
see proposal